FirebaseConfig/stagedev/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>368901996753-ltbuu6phjaa8uv2h7gumssh1mkpl3f7s.apps.googleusercontent.com</string>
```

Comment:
Please remove this tag

Reply:
@manishpathak3173 I've made the changes , Please review again.